data/outreachy.yml

```diff
@@ -299,3 +299,91 @@ rounds:
+
+
+          Creative coding is a type of computer programming that focuses on
+          generating artistic, expressive, and creative outputs using software
+          and digital tools. It has its applications in places such as game
+          development. Above all, it is a great pedagogical tool that gives
+          visual outputs to its readers.
+
+
+          Joy is a tiny creative coding library in Python. Joy builds heavily on
+          functional programming concepts with very little reference to Python
+          syntax.
+
+
+          This project aims to implement a geometric creative coding library in
+          OCaml. It is heavily inspired by Joy. When done, it will serve as a
+          means to do geometric creative coding in OCaml.
+        mentee: "Fay Carsons"
+        source: https://github.com/Sudha247/ocaml-joy
+        blog: "https://www.faycarsons.xyz"
+        mentors:
+          - Sudha Parimala
+          - Kaustubh M
+        video: https://watch.ocaml.org/w/b7sv1LQSVZQH6trf4xpwFX?start=3m8s
+      - title: "Implement a Dark Mode for OCaml.org"
+        description: >
+          OCaml is a powerful, statically-typed programming language known for
+          its efficiency and expressiveness. OCaml.org serves as the central hub
+          for the OCaml community, providing resources, documentation, and news.
+          In today's digital age, users expect a more personalised and
+          comfortable web experience. One such expectation is the availability
+          of a dark mode, which has become a popular feature on websites and
+          applications. This project outlines the plan to implement a dark mode
+          for OCaml.org, enhancing user experience and modernising the platform.
+          As OCaml continues gaining traction in various industries, it is
+          essential to modernise its online presence to meet users' expectations
+          worldwide.
+
+
+          The current styles and colors for light mode already exist so
+          implementing a dark mode will involve adding contrasting colors and
+          styles according to the Figma design. It will also consider accessibility
+          standards and create a button that toggles between the light
+          and dark mode.
+        mentee: "Oluwaseun Oyenuga"
+        source: https://github.com/ocaml-ppx/ppxlib
+        blog: "https://dev.to/burnleydev"
+        mentors:
+          - Sayo Bamigbade
+          - punchagan
+        video: https://watch.ocaml.org/w/b7sv1LQSVZQH6trf4xpwFX?start=21m12s
+      - title: "Improve the GUI Experience for OCaml Users"
+        description: >
+          Inspired by [Rust's "Are we GUI yet?"](https://areweguiyet.com/), we want the same work done on the
+          OCaml GUI libraries. A similar work has been done for the OCaml web
+          libraries: ["Is OCaml web yet?"](https://ocaml.org/docs/is-ocaml-web-yet) (see the pull request). This work would
```

Comment:
I can only find a pull request for "Is OCaml GUI yet?"

Reply:
Ah, I was copying the descriptions from the original Outreachy pages so I think they predate "Is OCaml web yet?" being merged! https://ocaml.org/docs/is-ocaml-web-yet :))

Reply:
Thanks for the editing @christinerose -- In 2fc2173 I made the descriptions support markdown so these links you added would be rendered on the web page :))